### ISSUE TITLE
Port Examples from Python Cookbook

### ISSUE THREAD

**Issue by h0rv** [instructor-ai/instructor-go#14]
https://python.useinstructor.com/examples/

**Comment by apvar** [comment]
Should these go in /examples ?

**Comment by h0rv** [comment]
@apvar correct, like `examples/<example_name>/main.go`. Then you can also add them to the README for documentation.

**Comment by apvar** [comment]
@h0rv Awesome!  Will make PR soon!
